src/container/home/CityCarousel.tsx

```diff
@@ -0,0 +1,83 @@
+import { Box, Card, BoxProps, Typography, Grid } from '@mui/material';
+import Image from '../../components/Image';
+import berlin from '../../images/berlin.jpeg';
+import bogota from '../../images/bogota.jpeg';
+import sydney from '../../images/sydney.jpeg';
+import taipei from '../../images/taipei.jpeg';
+
+type ItemProps = {
+  id: string;
+  city: string;
+  image: string;
+  latlon: number[];
+};
+
+interface Props extends BoxProps {
+  title?: string;
+  subheader?: string;
+}
+
+const cities = [
+  {
+    id: '1',
+    city: 'Berlin',
+    image: berlin,
+    latlon: [52.5170365, 13.3888599]
+  },
+  {
+    id: '2',
+    city: 'Bogota',
+    image: bogota,
+    latlon: [4.6534649, -74.0836453]
+  },
+  {
+    id: '3',
+    city: 'Sydney',
+    image: sydney,
+    latlon: [-33.8698439, 151.208284]
+  },
+  {
+    id: '4',
+    city: 'Taipei',
+    image: taipei,
+    latlon: [25.0375198, 121.5636796]
+  }
+];
+
+export default function CityCarousel({ sx }: Props) {
+  return (
+    <Grid item xs={12}>
```

Comment:
I think this is the container <Grid> , right?

so       `<Grid container>`

Reply:
To determine the spacing between grid items, you can also pass the follow prop to the container: `spacing={5}`